docker-compose.yml

```diff
@@ -47,26 +47,42 @@ services:
   # postgres
   postgres:
     image: docker.io/postgres:16-alpine
+    healthcheck:
+      test: pg_isready -d db_prod
```

Comment:
This is logging `[284] FATAL:  role "root" does not exist`.
I thin this would be better: `pg_isready -q -U netbox -d netbox`